File: py/IEC62325/MarketOperations/MktDomain/PassIndicatorType.py
# Converted by an OPENAI API call using model: gpt-3.5-turbo-1106 on Thu Dec 28 15:17:19 2023
from enum import Enum

class PassIndicatorType(Enum):
    """
    Defines the individual passes that produce results per execution type/market
    type.
    """
    MPM_1 = 1  # Market Power Mitigation Pass 1
    MPM_2 = 2  # Market Power Mitigation Pass 2
    MPM_3 = 3  # Market Power Mitigation Pass 3
    MPM_4 = 4  # Market Power Mitigation Pass 4
    RUC = 5  # Residual Unit Commitment
    RTPD = 6  # Real Time Pre Dispatch
    RTED = 7  # Real Time Economic Dispatch
    HA_SCUC = 8  # Hour Ahead Security Constrained Unit Commitment
    DA = 9  # Day Ahead

File: java/IEC62325/MarketOperations/MktDomain/PassIndicatorType.java
package IEC62325.MarketOperations.MktDomain;


/**
 * Defines the individual passes that produce results per execution type/market
 * type.
 * @created 03-Jan-2024 2:12:37 PM
 */
public enum PassIndicatorType {
	/**
	 * Market Power Mitigation Pass 1
	 */
	MPM-1,
	/**
	 * Market Power Mitigation Pass 2
	 */
	MPM-2,
	/**
	 * Market Power Mitigation Pass 3
	 */
	MPM-3,
	/**
	 * Market Power Mitigation Pass 4
	 */
	MPM-4,
	/**
	 * Residual Unit Commitment
	 */
	RUC,
	/**
	 * Real Time Pre Dispatch
	 */
	RTPD,
	/**
	 * Real Time Economic Dispatch
	 */
	RTED,
	/**
	 * Hour Ahead Security Constrained Unit Commitment
	 */
	HA-SCUC,
	/**
	 * Day Ahead
	 */
	DA
}